test/nightly/global.nightly.test.ts

```diff
@@ -5,24 +5,30 @@
 
 import { WebSiteManagementClient } from '@azure/arm-appservice';
 import { ResourceManagementClient } from '@azure/arm-resources';
-import { createTestActionContext, TestAzureAccount } from '@microsoft/vscode-azext-dev';
-import * as vscode from 'vscode';
+import { createTestActionContext, type TestActionContext } from '@microsoft/vscode-azext-dev';
 import { AzureAccountTreeItemWithProjects, createAzureClient, ext } from '../../extension.bundle';
 import { longRunningTestsEnabled } from '../global.test';
 
-export let testAccount: TestAzureAccount;
+import { createSubscriptionContext, subscriptionExperience, type ISubscriptionContext } from '@microsoft/vscode-azext-utils';
+import { type AzureSubscription } from '@microsoft/vscode-azureresources-api';
+import * as vscode from 'vscode';
+
 export let testClient: WebSiteManagementClient;
+let testContext: TestActionContext;
```

Comment:
Nit: I prefer removing this shared test context and going with how we initiated the two separately before with `createTestActionContext()`.  It's more in line with how we usually keep set up contexts for separate processes